pkgs/by-name/st/streamrip/package.nix

```diff
@@ -2,10 +2,8 @@
   lib,
   python3Packages,
   fetchFromGitHub,
-
   ffmpeg,
 }:
-
```

Comment:
Don't remove these blank lines. The first one separates the runtime dependencies from the ordered-by-appearance list of initial inputs (with `lib` being first by convention). The second one separates all inputs from the call to the function that creates the derivation.